tests/unit_tests/test_wave_06.py

```diff
@@ -76,16 +76,30 @@ def test_swap_best_by_category():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
+    # raise Exception("Complete this test according to comments below.")
     # *********************************************************************
     # ****** Complete Assert Portion of this test **********
     # *********************************************************************
     # Assertions should check:
     # - That the results is truthy
     # - That tai and jesse's inventories are the correct length
     # - That all the correct items are in tai and jesse's inventories, including the items which were swapped from one vendor to the other
+    assert result
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    #assert (item_a, item_b, item_c) in tai.inventory?? is there a way to not repeat yourself?
+    assert item_a in tai.inventory
+    assert item_b in tai.inventory
+    assert item_f in tai.inventory
+    assert item_d in jesse.inventory
+    assert item_e in jesse.inventory
+    assert item_c in jesse.inventory
```

Comment:
If we wanted to enforce the order, we could make these assertions shorter by saying

```
assert tai.inventory == [item_a, item_b, item_f]
```

If we didn't care about order, we knew there would not be any duplicates, and were sure all the items would be hashable, we could write:

```
assert set(tai.inventory) == {item_a, item_b, item_f}
```

But in general, sometimes tests get very repetitive, and that's OK! We're slightly less worried about repetition in our tests than we are in our program logic.